### DISCUSSION TITLE
Has anyone received NV Energy or Las Vegas Valley Water District meter readings

### DISCUSSION THREAD

**Discussion by chibiconsulting** [bemasher/rtlamr#311] [General]
First off , thanks @bemasher for a great bit of software.
I live in the Las Vegas, NV area and after setting up rtlamr it immediately started receiving gas company meters on the default 912Mzh setting with SCM after testing all message types, it now gets both SCM & SCM+ consistently receiving 25 - 30 gas meters or so, at about two times a minute. Was able to find mine right away in the flood of data. Wow! Cool! This will be useful in the winter.
Has anyone found either NV Energy or Las Vegas Valley Water District meters? If so, what frequency / message types?

**Comment by lvilluminati** [comment]
Hi there, funny enough I live in the Vegas area as well, on the Southwest side. Mostly what I receive is water meters (the IDs usually start with a 7). That one is definitely SCM+, never seen SCM packets. I think my gas meter only transmits when it's being read (wake up I believe it's called), I haven't seen one yet but my meter reading date is today so I may see one. I do get a ton of readings for device type 12 which does correspond to gas meters, but none of the device IDs are anywhere close to mine. Almost all the readings I get for type 12 the ID starts with a 9, whereas mine starts with a 2. I have only been running RTLAMR for about a month so like I said I am still waiting to see if my gas readings are triggered.

I'm pretty sure you will not see any from your NVEnergy meter, I BELIEVE (not totally sure) that Sensus/FlexNet meters use a proprietary and encrypted network to communicate, although I couldn't find whether it's RF based or PLC (data over power lines) based. I have read a few places that meter readings are sent every 15 minutes so if they were using a standard ERT protocol, you should see tons of them.

**Comment by chibiconsulting** [comment]
Interesting. I guess that depending on when each neighborhood is built, it gets a different combination of Electric/Gas/Water meters with different transmitter capabilities.  I am getting both SCM and SCM+ for Gas and very little for Water & Electric. Are all of the meters you have found been on the default frequency center of 912600155? I am wondering if the Water or Electric are on a different frequency.

For reference, here is a few days summary for frequency center of 912600155 near my house. (table separator "|") *** rtlamr flag of -unique=true to only count new meter readings, or the count totals would be ridiculous.

Times Counted | Meter Ids | Format | Type
1 | 3XXXXXX | SCM | 13 Water
84 | 4XXXXXX | SCM | 12 Gas
1 | 5XXXXXX | SCM+ | 0x28 Electric
45 | 6XXXXXX | SCM | 9 Gas
48 | 7XXXXXX | SCM+ | 9 Gas
37 | 9XXXXXX | SCM+ | 9 Gas

Type reference:
    Electric: 04, 05, 07, 08
    Gas: 02, 09, 12
    Water: 11, 13
For SCM+, Least significant nibble is equivalent to SCM's endpoint type field. So 0x28 = 8, 0x9c = 12, 0xAB = 11.

The one type 13 Water and one type 0x28(8) Electric seem to be the exceptions for me. Maybe a data error on my part.

**Reply by lvilluminati** [reply]
Yes, I get mostly type 11 (Water) readings, almost all with an ID beginning with 7 (which correlates to my water meter), and a lot of type 12 (Gas) readings, mostly starting with 9 (which does not correlate with my gas meter), but never my own meter that has an ID starting with 2 (at least as it's printed on the gas meter). I have not tried being a weirdo and looking at my neighbors gas meter IDs to see if theirs are similar :sweat_smile: 

Maybe I'll try listening to SCM for a while to see if I receive anything, but last time I tried, I got nothing.

Interesting you do not get any water meter readings. Have you tried opening the water utility cover on your sidewalk to look at your ERT meter ID?

Also, I have a few types with 1 reading that I assume are noise that have nonsense IDs or other data, but somehow passed the checksum. I also have a couple that make more sense, but have an ID length of 9 instead of the usual 8.  Here are all the device types I have received so far:

| DeviceType | Count |
| --- | --- |
| 0x9c | 171107 |
| 0xab | 13772 |
| 0xdc | 2 |
| 0xbc | 3 |
| 0x5d | 1 |
| 0xeb | 1 |
| 0xf1 | 1 |
| 0x52 | 1 |
| 0x98 | 1 |
| 0xed | 1 |
| 0x49 | 1 |

**Reply by lvilluminati** [reply]
Actually, looking at this again, it appears I get mostly gas meters... just never mine 😆

**Comment by chibiconsulting** [comment]
* Yeah, the number on the water meter is the same as on the water bill, the id starts with 1XXXXXX, no matches at all. * disregard this part. 
 
Inserted 2026-09-02: For me at least, is that the meter id has nothing to do with any id listed on the LVVWD account.

** Starting some testing in the 460Mhz range as the LVVWD has some frequencies in that range, but no idea if that will show anything, or how long that will take. ** disregard this part also, water meter is in 915 band, not in 460 range.

If you set the message type to -msgtype=scm,scm+ you will get both type of meter readings.

Regarding your Gas meter, what type of antenna are you using? If all the meters you read are further away, maybe your signal is too powerful. Try a different antenna. {or location.} Also, there is a gain setting in the rtlamr configuration, [-gainbyindex=0: set gain by index]. Unknown what index values might be useful.

**Comment by bcarpio** [comment]
Reviving this — also Las Vegas, northwest side (89143), LVVWD.

Did either of you ever get clean LVVWD water meter decodes? Specifically:

What center frequency and message type (SCM, SCM+, IDM, R900)?
Did the consumption field track actual usage, or did the payload look like garbage / obviously encrypted?
Roughly how far was your dongle from the meter pit, and what antenna?

Asking because my Flume 2 just died and I'd rather read the utility meter directly than buy another one. LVVWD is also mid-rollout replacing mechanical meters with ultrasonic AMI units, so I'm curious whether anyone here has seen a post-swap meter and whether it still bubbles up in ERT.

I still have a mechanical meter for now. Happy to report back with results if I go ahead.

**Reply by chibiconsulting** [reply]
* _No. not been able to receive any good LVVWD water meter data at all._ * disregard, see below.
Lots of gas meter readings, which I can use as a partial proxy of hot water use...
The gas meter readings are fairly robust, used SCM,SCM+ setting and frequency of 912600155 will spit out tons of data from all sorts of gas meters, just not any useful LVVWD data.

**Reply by chibiconsulting** [reply]
Update, after capturing all available data to a database for a while, it seems that I had forgotten the specifications. It turns out that, yes; type 11 and type 0xAB can be used for water meters. The problem with LVVWD, for me at least, is that the meter id has nothing to do with any id listed on the LVVWD account. It also turns out that the reading value on the monthly billing is [ reading / 100 ] so after doing that conversion, the usage figures lined up and I found my water meter.

**Reply by bcarpio** [reply]
How long did you have to listen until the water meter checked in? I picked up one of these: 
SDR-V4 Blog RTL SDR V4 R828D RTL2832U 1PPM TCXO SMA RTLSDR Software Defined Radio Receiver, With Multipurpose Antenna, but all I can see are gas meters so far.

**Reply by chibiconsulting** [reply]
In terms of how long, it depends on how much you are using the water; during a shower, doing laundry or washing the dishes, it only takes a few minutes. If you are not using much water, it could take a day or more.

The key for me was that I was incorrectly  looking for my LVVWD account number; and that is not what they used to identify it. Once I found the correct meter id, I searched though old logs and found it was always there, just buried in with everyone else's data.

Search using both SCM, and SCM+ in RTLAMR and look for any readings other than the one your gas meter is using, water is type 0xAB in my case; then put them in a spread sheet and compare to your water bills, one should pop right out.

Also, you do not really need a multipurpose antenna; any antenna for the 915MHz ISM band will work. Look for ISM / Meshtastic / LoRa when searching for antennas, there are a lot of inexpensive ones out there.

**Reply by bcarpio** [reply]
I was able to figure out the issue, my cast iron cover, or the city's cast iron cover was blocking my Radio Receiver. I remove the cover temporarily, put a rubber bumper plate over it (25lb I use for weight lifting) so no one accidentally fell in while I was testing and the receiver started picking up the signal right away. 

So the issue for me at the moment is this cast iron cover over the meter.

**Reply by chibiconsulting** [reply]
Makes sense. Try an antenna for the 915MHz ISM band.

The water company is able to read it in real time, so the cast iron is not stopping them; and they probably use one that is tuned for the 915Mhz band.
